src/transform/FooterContainer.css

```diff
@@ -0,0 +1,25 @@
+
+.footer_container {
```

Comment:
It's fine to punt on this but consider using a common prefix like pagelib-footer-thing.